recipe/meta.yaml

```diff
@@ -1,49 +1,47 @@
-{% set version = "3.6.0" %}
-{% set name = "tinydb" %}
-{% set sha = "64e6bc11c854ecc9ff7db2d0a26869ba47d9ebf79620260e40ae8ef611163c41" %}
+{% set version = "3.8.1.post1" %}
 
 package:
-    name: {{ name }}
-    version: {{ version }}
+  name: tinydb
+  version: {{ version }}
 
 source:
-    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    fn: tinydb-{{ version }}.tar.gz
-    sha256: {{ sha }}
+  url: https://pypi.io/packages/source/t/tinydb/tinydb-{{ version }}.tar.gz
+  sha256: 961d7fbeb09843251b8a016965269c2fb0e898f16e8686bcf40e23d385219d78
 
 build:
-    number: 0
-    script: python setup.py install --single-version-externally-managed --record=record.txt
-    noarch: python
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
-    build:
-        - setuptools
-        - python
-    run:
-        - python
+  build:
+    - pip
+    - python
+    - pytest-runner
```

Comment:
@ericmjl this is the important change pytest-runner is listed as a setup-requires in setup.py.

Reply:
:thumbsup: Thanks @ocefpaf!